lib/src/timeline.dart

```diff
@@ -75,7 +75,6 @@ class Timeline {
   bool _collectHistoryUpdates = false;
 
   bool get canRequestHistory {
-    if (events.isEmpty) return true;
     return room.prev_batch != null && events.last.type != EventTypes.RoomCreate;
```

Comment:
You can only access events.last, if events is not empty.

Reply:
for the normal room history, events should never have to be empty, since we should always be able to request our own history for our join event at least. This also might cause issues, since possibly on the first history pagination a timeline has no prev_batch token and is empty, if you just want to request the history for an arbitrary room? Not sure.

Reply:
okay true, somehow I have missread the whole method. Will close this then